packages/caliper-core/lib/report/report.js

```diff
@@ -45,49 +45,55 @@ class Report {
     createReport(absConfigFile, absNetworkFile, blockchainType) {
         let config = CaliperUtils.parseYaml(absConfigFile);
         this.reportBuilder.addMetadata('DLT', blockchainType);
-        try{
```

Comment:
the caliper-core report.js has been modified by the editorconfig settings you are using. I'm a fan of these settings, but we should apply them globally through the entire project in a separate PR - is it possible to reverse the changes in this file?

Reply:
Solved :)